src/Form/SsofactRegisterForm.php

```diff
@@ -111,8 +111,9 @@ public function buildForm(array $form, FormStateInterface $form_state) {
     ];
     $form['redirect_url'] = [
       '#type' => 'hidden',
-      // @todo Use current path.
-      '#value' => Url::fromUri('internal:/')->toString(),
+      // Redirect to user account dashboard after clicking link in confirmation
+      // email.
+      '#value' => Url::fromUri('internal:/shop/user/account', ['absolute' => TRUE])->toString(),
```

Comment:
@juliquiron This change actually needs to be performed for the user registration form used for the 1-article-test, too…

I assume we need to rebase this branch onto paywall-juli then?  But if we do that, this fix cannot be merged into master, where it is needed as soon as possible.  Therefore, I would suggest that I additionally apply this change for the new form directly in #3 ?